### PR TITLE
Sensor locatin4

### PR DESCRIPTION
**Pull request by tzachi-dar**
Against latest head.

### CODE REVIEW

File: app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        buttonCancel = (Button)findViewById(R.id.saveSensorLocationCancel);
+        sensor_location_other = (EditText) findViewById(R.id.edit_sensor_location);
+        sensor_location_other.setEnabled(false);
+        DontAskAgain = (CheckBox)findViewById(R.id.sensorLocationDontAskAgain);
+        radioGroup = (RadioGroup) findViewById(R.id.myRadioGroup);
+        addListenerOnButton();
+        
+        locations = new LinkedList<Location>();
+        
+        locations.add(new Location("I dont wish to share", PRIVATE_ID));
```
Comment (jessiepusateri) [inline app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java]: Can you change this to either `I do not wish to share` or `I don't wish to share`?

File: app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        locations.add(new Location("Upper arm", 1));
+        locations.add(new Location("Thigh", 2));
+        locations.add(new Location("Belly (abdomen)", 3));
+        locations.add(new Location("Lower back", 4));
+        locations.add(new Location("buttocks", 5));
```
Comment (jessiepusateri) [inline app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java]: Can you change this from `buttocks` to `Buttocks` to match the case of the rest of the options?

File: app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        locations.add(new Location("other", OTHER_ID));
```
Comment (jessiepusateri) [inline app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java]: Can you change this from `other` to `Other` to match the case of the rest of the options?

File: app/src/main/res/layout/activity_new_sensor_location.xml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                android:gravity="center_horizontal">
+
+
+
+                    <TextView
+                        android:id="@+id/sensor_location_text"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:gravity="left"
+                        android:text="In the future xDrip would like to send anonymous sensor data to the developers to improve the calibration algorithm... If you decide to opt-in, you will be asked to confirm before each data transfer. Data will never be sent without your permission. To help us understand and fine tune the calibration we would like to know where this sensor has been placed. Press the back button to cancel."
```
Comment (jessiepusateri) [inline app/src/main/res/layout/activity_new_sensor_location.xml]: Can you change `fine tune` to `fine-tune`?

**Comment by StephenBlackWasAlreadyTaken** [comment]
@tzachi-dar it looks like the db version bump got left out of this PR

**Comment by StephenBlackWasAlreadyTaken** [comment]
(If someone was running beta3 and updates to this they will crash)

**Comment by tzachi-dar** [comment]
I have increased the version to 36 and added the
file app\src\main\assets\migrations\36.sql.
meanwhile on androidmanfest the version is already 36, but there is no
36.sql file.

Should I rename my change 37 and add rename the file to 37.sql and change
the androidmanfest version?

Thanks
Tzachi

**Comment by StephenBlackWasAlreadyTaken** [comment]
Yeah but unfortunately that will make yours crash, so for you personally you need to first just increment the manifest (and not the migration) and install that on your phone. And then increment the .sql file and install that on your phone (and put up the pr for both of those changes together)

The active android migrations definitely leave something to be desired!

**Pull request merged**